zulipterminal/ui_tools/boxes.py

```diff
@@ -200,11 +200,11 @@ def __init__(self, message: Message, model: Any,
                  last_message: Any) -> None:
         self.model = model
         self.message = message
+        self.message['links'] = OrderedDict()
```

Comment:
Currently we treat the message data as essentially the data raw from the API. I'm a little concerned about adding extra data there since we don't know if the server may add a `links` data structure for messages, which this will then override.